sdk/eventhub/event-hubs/test/README.md

```diff
@@ -2,19 +2,18 @@
 
```

Comment:
Line 35...

> assign the **Owner** role to the registered application

Owner role won't give us the permissions.. **Azure Event Hubs Data Owner** role I think.

Reply:
Am guessing a similar change would apply to Service Bus as well? Can you pretty please share the text would recommend? Or is it just changing `Owner` to `Azure Event Hubs Data Owner` and `Azure Service Bus Data Owner`?

Reply:
Just changing `Owner` to `Azure Event Hubs Data Owner`.
Service Bus is already updated.

Reply:
Done https://github.com/Azure/azure-sdk-for-js/pull/8602/commits/e6df7be56e6e81a0441a776538e44d3123e3ccf6

Reply:
Fantastic catch.